cypress/e2e/tests/pages/about.spec.ts

```diff
@@ -0,0 +1,165 @@
+import HomePagePo from '@/cypress/e2e/po/pages/home.po';
+import BurgerMenuPo from '@/cypress/e2e/po/side-bars/burger-side-menu.po';
+import AboutPagePo from '@/cypress/e2e/po/pages/about.po';
+import DiagnosticsPagePo from '@/cypress/e2e/po/pages/diagnostics.po';
+import * as path from 'path';
+
+const aboutPage = new AboutPagePo();
+const downloadsFolder = Cypress.config('downloadsFolder');
+
+describe('About Page', { testIsolation: 'off', tags: ['@adminUser', '@standardUser'] }, () => {
+  before(() => {
+    cy.login();
+  });
+
+  it('can navigate to About page', () => {
+    HomePagePo.goToAndWaitForGet();
+
+    const burgerMenu = new BurgerMenuPo();
+
+    BurgerMenuPo.toggle();
+    burgerMenu.about().click();
+    aboutPage.waitForPage();
+  });
+
+  it('can navigate to Diagnostics page', () => {
+    aboutPage.goTo();
```

Comment:
i think we should be able to start using the `navTo` concept now. So instead of doing a costly page load every time the page is reached by user clicks.

so something similar to the first test, but in about.po
```
  static navTo() {
    BurgerMenuPo.toggle();
    burgerMenu.about().click();
    aboutPage.waitForPage();
  }
```

also applies to the goTo's below (if the page is still in dashboard world)

Reply:
I was able to apply this change anywhere there was a goTo and the page is still in dashboard world except for the CLI Downloads - those tests fail when this change is applied so they're still using goTo's